ynput/ayon-core
# Maya: Enhancement on reset frame range based on fps

## Changelog Description
- Support for fps options in terms of maya settings(such as 2, 3, 4, 5, 6, 8, 10, 12, 16 fps)
- Support to set fps value based on the task

## Additional info
n/a

## Testing notes:
1. Go to ayon server -> Project Setting -> project and select your project
2. Set the fps value based for your task
![image](https://github.com/ynput/ayon-core/assets/64118225/04debc48-9f16-4e94-890e-835a8ec0e99e)
3. Launch Maya
4. Access AYON menubar -> Set frame range
5. Should be set the fps value according to the project setting.
